sources/packages/web/src/components/aest/students/ApplicationHeaderTitle.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <detail-header :header-map="headerMap" />
+</template>
+
+<script lang="ts">
+import { defineComponent, onMounted, ref } from "vue";
+import { ApplicationSupplementalDataAPIOutDTO } from "@/services/http/dto";
+import { ApplicationService } from "@/services/ApplicationService";
+import DetailHeader from "@/components/generic/DetailHeader.vue";
+import { useFormatters } from "@/composables";
+
+export default defineComponent({
+  components: { DetailHeader },
+  props: {
+    applicationId: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup(props) {
+    const applicationData = ref<ApplicationSupplementalDataAPIOutDTO>();
+    const headerMap = ref<Record<string, string>>({});
+    const { dateOnlyLongString } = useFormatters();
+
+    const mapApplicationHeader = (
+      application: ApplicationSupplementalDataAPIOutDTO,
+    ): Record<string, string> => {
+      if (
+        application.applicationNumber &&
+        application.applicationInstitutionName
+      ) {
+        if (
+          application.applicationStartDate &&
+          application.applicationEndDate &&
+          application.applicationOfferingIntensity
+        ) {
+          return {
+            Name: application.studentFullName,
+            "Application number": application.applicationNumber,
+            Institution: application.applicationInstitutionName,
+            "Study dates": `${dateOnlyLongString(
+              application.applicationStartDate,
+            )} - ${dateOnlyLongString(application.applicationEndDate)}`,
+            Type: application.applicationOfferingIntensity,
+          };
+        }
+
+        return {
+          Name: application.studentFullName,
+          "Application number": application.applicationNumber,
+          Institution: application.applicationInstitutionName,
+        };
+      }
```

Comment:
We can simplify the logic by slightly adjusting the `DetailHeader` to NOT display a map if the value is not present. 

![image](https://github.com/user-attachments/assets/ea8c581d-861e-4787-a2d1-6cee1cf21a57)

Reply:
Edited the solution of DetailHeader screenshot

Reply:
updated